specification/communication/data-plane/Microsoft.CommunicationServicesAdministration/ga/2021-03-07/phonenumbers.json

```diff
@@ -0,0 +1,932 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Phone Number Administration Service",
+    "description": "Azure Communication Service",
+    "contact": {
+      "email": "acsdevexdisc@microsoft.com"
+    },
+    "version": "2021-03-07"
+  },
+  "paths": {
+    "/phoneNumbers/{phoneNumber}": {
+      "get": {
+        "tags": [
+          "AcquiredNumbers"
+        ],
+        "summary": "Gets the details of the given acquired phone number.",
+        "operationId": "PhoneNumber_GetByNumber",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "phoneNumber",
+            "description": "The acquired phone number whose details are to be fetched. E.g. +11234567890.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/AcquiredPhoneNumber"
+            }
+          },
+          "default": {
+            "description": "Failure",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Fetch acquired phone number": {
+            "$ref": "./examples/GetAcquiredPhoneNumberAsync.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Release"
+        ],
+        "summary": "Creates a release request for the given phone number.",
+        "operationId": "PhoneNumber_Release",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "phoneNumber",
+            "description": "Phone number to be released. E.g. +11234567890.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "204": {
```

Comment:
Response is missing schema:
"schema": {
  "$ref": "#/definitions/AcquiredPhoneNumber"
}